llvm/test/CodeGen/AMDGPU/v_swap_b16.ll

```diff
@@ -0,0 +1,39 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; RUN: llc -march=amdgcn -mcpu=gfx1100 -verify-machineinstrs < %s | FileCheck -check-prefixes=GFX11 %s
```

Comment:
It looks like v_mov_b16 and v_swap_b16 need to have the UseRealTrue16Insts predicate applied to them. We don't want to select them unless passing the feature flag mattr=+real-true16.

Reply:
done

Reply:
I will merge GFX12 v_swap_b16 mc change later in another patch

Reply:
The gfx12 instructions were added in this patch, so the tests should also be. 

In addition to passing the mattr=+real-true16 in the test, we need a behavior change to emitted code and another test, of the default behavior. 
v_mov_b16 and v_swap_b16 need to have the UseRealTrue16Insts predicate applied to them in tablegen. We don't want to select them unless passing the feature flag mattr=+real-true16. 

We don't want to use any real true16 instruction by default. 
We should have 2 runlines for every 16 bit test. 1 with +real-true16, and one with no attribute specified, to check the default behavior (which is -real-true16).  In this case, it was clear in the prior version of the patch when the runline had no attribute specified the v_swap_b16 was still emitted.

Reply:
Thanks for the comment! Just trim the patch to MC only and added GFX12 mc test